index.js

```diff
@@ -70,5 +76,52 @@ function toggleColor(element) {
 
 /***** Deliverable 2 *****/
 
+const likeBtn = document.querySelector(".like-button")
+const likes = document.querySelector(".likes")
+
+
+function increaseLikes() {
+  const newLikes = parseInt(likes.textContent) + 1
+  likes.textContent = `${newLikes} Likes`
+}
+
+
+likeBtn.addEventListener("click", increaseLikes)
+
 
 /***** Deliverable 3 *****/
+
+// create variable for form, link and description
+const form = document.querySelector("#new-goal-form")
+
+
+
+// const goalId = player.goals.length + 1
+// haha i thought i was gonna be all clever with player.goals.length + 1 as the id -- but the input data doesn't persist. Thus all my new links would have a goal id of 3 lolol. back to the drawing board!
+
+const ul = document.querySelector("ul")
+const goalId = ul.children.length +1
+// boo this didn't work either. All of my new data-ids are 3.... why is that?
```

Comment:
They're always going to be 3 because you're updating the goalId outside of the eventListener, so line 103 only runs 1 time - when the file is loaded and goalId is forever set to 3. Add line 103 inside of your eventListener instead, so that it runs each time the form is submitted.